### PR TITLE
Add KELOD Banner via $wgSiteNotice

### PR DESCRIPTION
**Pull request by deer-wmde**
Depends on https://github.com/wbstack/mediawiki/pull/427

https://phabricator.wikimedia.org/T357667

Adds the KELOD Research banner via $wgSiteNotice, with a time constraint until March 24th.

### CODE REVIEW

File: dist-persist/wbstack/src/Settings/LocalSettings.php [hunk 1]
```diff
@@ -495,6 +495,22 @@ function onSkinTemplateNavigationUniversal( SkinTemplate $skin, array &$links )
 wfLoadExtension( 'Score' );
 $wgMusicalNotationEnableWikibaseDataType = true;
 
+# DismissableSiteNotice - https://www.mediawiki.org/wiki/Extension:DismissableSiteNotice
+wfLoadExtension( 'DismissableSiteNotice' );
+$wgMajorSiteNoticeID = 1;
+$wgDismissableSiteNoticeForAnons = true;
```
Comment (m90) [inline dist-persist/wbstack/src/Settings/LocalSettings.php:501]: Should this be moved into the `if` condition below as well? Or does it cause build problems?

Comment (deer-wmde) [reply dist-persist/wbstack/src/Settings/LocalSettings.php:501]: It does not cause build problems I think. Not sure what I prefer, probably a good idea to only load what we need though

File: dist-persist/wbstack/src/Settings/LocalSettings.php [hunk 2]
```diff
@@ -495,6 +495,22 @@ function onSkinTemplateNavigationUniversal( SkinTemplate $skin, array &$links )
...
+
+// KELOD research banner campagin 2024 Q1 - https://phabricator.wikimedia.org/T357667
+// Visible until March 24th 2024 00:00:00 UTC)
+if (time() < mktime(0, 0, 0, 3, 24, 2024)) {
+    $wgSiteNotice = <<<EOF
+    <div style="width:98%; border:3px solid #0566C0; overflow:hidden; background-color: #F9F9FF; padding:16px 16px 16px 16px">
+    <div style="text-align:left; font-size:1.5em; color: #0566C0">Participants for knowledge equity project needed</div>
+    <div style="text-align:left;">Help us better understand how Wikidata, Wikibase Suite, and Wikibase Cloud support and pose barriers to knowledge equity. We would like you to participate if you hold and contribute historically marginalized knowledge, using any of these products. If interested, please '''follow the link to fill out the survey →''' [https://meta.wikimedia.org/wiki/Wikimedia_Deutschland/Knowledge_Equity_in_Linked_Open_Data_Research Knowledge Equity in Linked Open Data project]</div>
```
Comment (m90) [inline dist-persist/wbstack/src/Settings/LocalSettings.php:509]: I'm not a product person, but as a user I would prefer the link to be of `target="_blank"`. No need to act upon this comment unless it sparks interest.

**Review by m90 — APPROVED** [review]
Tested this locally and it works as advertised. No techinal objections against merging this from my end.

File: dist-persist/wbstack/src/Settings/LocalSettings.php (hunk 2 above)
Comment (m90) [reply dist-persist/wbstack/src/Settings/LocalSettings.php]: Could we just raw HTML our way into what we want instead?

**Pull request merged**